tests/devices/unit_tests/test_tetramm.py

```diff
@@ -171,25 +171,6 @@ async def test_hints_self_by_default(tetramm: TetrammDetector):
     assert tetramm.hints == {"fields": [tetramm.name]}
 
 
-async def test_prepare_with_too_low_a_deadtime_raises_error(
```

Comment:
This test is ensuring that we don't try to run the TetrAMM too fast during flyscans and get weird datasets or timeouts at the end. I would really like to see the test restored and the TetrAMM prepare method (the device is defined in this repository) modified to ensure the existing behaviour.